oaiharvest/harvest.py

```diff
@@ -303,6 +304,8 @@ def main(argv=None):
         except Exception as e:
             # Log error
             logger.error(str(e))
```

Comment:
No, I was wrong - also documented in Python 2.6 and seems to work fine. I'll commit it